src/Components/Order.js

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = ({ orderType }) => {
+  const [orders, setOrders] = useState(0);
+  const OrderOne = () => {
```

Comment:
State and `onClick` function work very well together - well done! This is a really nice and clean component👏 
 I'm just missing some blank lines for better readability - after `useState` and after `orderOne` function🙂 also the function name should start with lower case

Reply:
done